log/logger.py

```diff
@@ -0,0 +1,37 @@
+import logging
+from os import path
+import json
+from sys import stdout
+
+
+class Logger:
+    def __init__(self):
+        current_dir_path = path.dirname(__file__)
+        config_path = path.join(current_dir_path, '../confd/config/default.json')
+        with open(config_path, encoding='utf-8') as config_file:
+            self.__config = json.loads(config_file.read())
+
+        logging.basicConfig(level=self.__config['logger']['level'],
+                            filename=self.__config['logger']['filename'],
+                            datefmt=self.__config['logger']['dateFormat'],
+                            format=self.__config['logger']['format'])
+        self.log = logging.getLogger(__name__)
+        self._create_file_handler()
+
+    def _create_file_handler(self):
+        if self.log.handlers:
+            self.log.handlers = []
+        file_handle = logging.FileHandler(r'F:/Repos/exporter-worker/src/ew_logs.log')
```

Comment:
hard coded - get from config
add log files to git ignore
Why don't you use winston logger? or some other infra logger?

Reply:
-Why don't you use winston logger? or some other infra logger? 
A: Python already have 'Logger' module built-in called 'logging' same as winston , very easy to use.

Reply:
fixed in c42b798